src/App.tsx

```diff
@@ -14,11 +14,85 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[];
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGoods: ['Jam'],
+  };
+
+  toggleAdd = (good: string) => {
+    if (good) {
+      this.setState((prevState) => {
+        return !this.isGoodIncluded(good)
+          ? { selectedGoods: [...prevState.selectedGoods, good] }
+          : { selectedGoods: prevState.selectedGoods.filter(item => item !== good) };
+      });
+    }
+  };
```

Comment:
How did you come up with it))

Reply:
> How did you come up with it))

it was late night and I thought, why use two buttons?

Reply:
And this is how, kids, I invented facebook...